### PR TITLE
shadowfire stone quest fixes

### PR DESCRIPTION
**Pull request by TectonicRifts**
Changes based on retail video and teaser screenshots:

https://www.youtube.com/watch?v=m9wIe5BOkAc
http://acpedia.org/wiki/File:Teaser_200602JackCat.jpg

- Spawn placement corrections in 006E dungeon landblock in portions not pcapped
- Filled section of dungeon at very end, which had been left empty
- Increased scale of elementals such as Luminary Knight and Incendiary Knight
- Reduced scale of Inner Chamber Guardian
- Inner Chamber Guardian now wields correct weapon
- Added 20 hour pickup timer to Empyrean Notebook
- Added text to Empyrean Notebook
- Fixed art of Barbaric Mukkir (added clothing base, corrected palette template to blue)

### CODE REVIEW

**Review by ziang4891 — DISMISSED** [review]

File: Database/Patches/2006-01-Reprisals/9 WeenieDefaults/Creature/Mukkir/31897 Barbaric Mukkir.sql [hunk 1]
```diff
@@ -61,6 +61,7 @@ VALUES (31897,   1,   33559741) /* Setup */
      , (31897,   3,  536871107) /* SoundTable */
      , (31897,   4,  805306368) /* CombatTable */
      , (31897,   6,   67116771) /* PaletteBase */
+     , (31897,   7,  268437061) /* ClothingBase */
      , (31897,   8,  100688542) /* Icon */
      , (31897,  22,  872415417) /* PhysicsEffectTable */
      , (31897,  30,         86) /* PhysicsScript - BreatheAcid */;
```
Comment (DridsGhost) [inline Database/Patches/2006-01-Reprisals/9 WeenieDefaults/Creature/Mukkir/31897 Barbaric Mukkir.sql:67]: Recommend

     , (31897,  35,        420) /* DeathTreasureType - Loot Tier: 6 */;
     
     INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
VALUES (31897,  6, 0, 3, 0, 394, 0, 0) /* MeleeDefense        Specialized */
     , (31897,  7, 0, 3, 0, 500, 0, 0) /* MissileDefense      Specialized */
     , (31897, 13, 0, 3, 0, 400, 0, 0) /* UnarmedCombat       Specialized */
     , (31897, 15, 0, 3, 0, 280, 0, 0) /* MagicDefense        Specialized */
     , (31897, 24, 0, 3, 0, 400, 0, 0) /* Run                 Specialized */
     , (31897, 31, 0, 3, 0, 270, 0, 0) /* CreatureEnchantment Specialized */
     , (31897, 33, 0, 3, 0, 270, 0, 0) /* LifeMagic           Specialized */
     , (31897, 34, 0, 3, 0, 270, 0, 0) /* WarMagic            Specialized */
     , (31897, 44, 0, 3, 0, 400, 0, 0) /* HeavyWeapons        Specialized */
     , (31897, 45, 0, 3, 0, 400, 0, 0) /* LightWeapons        Specialized */
     , (31897, 46, 0, 3, 0, 400, 0, 0) /* FinesseWeapons      Specialized */;

INSERT INTO `weenie_properties_body_part` (`object_Id`, `key`, `d_Type`, `d_Val`, `d_Var`, `base_Armor`, `armor_Vs_Slash`, `armor_Vs_Pierce`, `armor_Vs_Bludgeon`, `armor_Vs_Cold`, `armor_Vs_Fire`, `armor_Vs_Acid`, `armor_Vs_Electric`, `armor_Vs_Nether`, `b_h`, `h_l_f`, `m_l_f`, `l_l_f`, `h_r_f`, `m_r_f`, `l_r_f`, `h_l_b`, `m_l_b`, `l_l_b`, `h_r_b`, `m_r_b`, `l_r_b`)
VALUES (31897,  0,  4,  70, 0.75,  350,  242,  280,  210,  350,  350,  385,  350,    0, 1,  0.1,    0,    0,  0.1,    0,    0,  0.1,    0,    0,  0.1,    0,    0) /* Head */
     , (31897,  5,  1,  70, 0.75,  350,  242,  280,  210,  350,  350,  385,  350,    0, 2, 0.45,  0.2,    0, 0.45,  0.2,    0, 0.45,  0.2,    0, 0.45,  0.2,    0) /* Hand */
     , (31897, 16,  4,  70,  0.5,  350,  242,  280,  210,  350,  350,  385,  350,    0, 2, 0.45,  0.4, 0.45, 0.45,  0.4, 0.45, 0.45,  0.4, 0.45, 0.45,  0.4, 0.45) /* Torso */
     , (31897, 18,  2,  90,  0.5,  350,  242,  280,  210,  350,  350,  385,  350,    0, 2,    0,  0.2,  0.1,    0,  0.2,  0.1,    0,  0.2,  0.1,    0,  0.2,  0.1) /* Arm */
     , (31897, 19,  2,  90, 0.75,  350,  242,  280,  210,  350,  350,  385,  350,    0, 3,    0,  0.2, 0.45,    0,  0.2, 0.45,    0,  0.2, 0.45,    0,  0.2, 0.45) /* Leg */
     , (31897, 22, 32, 140,  0.5,    0,    0,    0,    0,    0,    0,    0,    0,    0, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0) /* Breath */;


INSERT INTO `weenie_properties_spell_book` (`object_Id`, `spell`, `probability`)
VALUES (31897,  2074,   2.02)  /* Gossamer Flesh */
     , (31897,  2146,   2.02)  /* Evisceration */
     , (31897,  2162,   2.02)  /* Olthoi's Gift */
     , (31897,  2164,   2.02)  /* Swordsman's Gift */
     , (31897,   285,   2.02)  /* Magic Vulnerability VI */;

**Comment by DridsGhost** [comment]
Currently Luminary knights 70974, are set to do 268435456 damage, which displays 
Luminary Knight hits your chest for 3 points of prismatic damage!
Your PR changes it to 4, bludge. I assume we reviewed PCAPs for that?

**Comment by DridsGhost** [comment]
Further conversation with Tectonic reveals some shadowfire weapons and recipes are missing. Along with the current slayer damage bonus being 3.

We will require a shard script update with the above before approving.

This will require damage tuning as well.

**Comment by ziang4891** [comment]
Hells confirmed that the Luminary Knight does bludgeoning damage https://discord.com/channels/261242462972936192/729525560421646469/758055571944374413

**Comment by ziang4891** [comment]
"Further conversation with Tectonic reveals some shadowfire weapons and recipes are missing." is incorrect

**Comment by DridsGhost** [comment]
Okay Ziang. Looked like we were missing the two hander, but upon further inspection I see it there. Also I missed the MoA info. Thanks. 

What about updating the slayer damage mod value? 3 seems to high.

**Comment by ziang4891** [comment]
SlayerBonus 3 is what the items have that are posted to LSD, and they've been in ACE this way since like March, or so.  No one that I know of has complained about it being too high.  It didn't seem excessive when I tested them on Shadows.

**Comment by DridsGhost** [comment]
Sounds good. I did some testing tonight at those values and it seems fine. Approving.

**Review by DridsGhost — APPROVED** [review]

**Review by ziang4891 — APPROVED** [review]

**Pull request merged**